crates/core/src/subscription/execution_unit.rs

```diff
@@ -54,7 +51,17 @@ impl QueryHash {
 #[derive(Debug)]
 pub struct ExecutionUnit {
     hash: QueryHash,
-    plan: SupportedQuery,
+    eval_plan: SupportedQuery,
```

Comment:
Do we need to hold both the eval and eval_incr plans? The eval plan will only be executed once at subscribe time.

Reply:
I assumed (but have not checked) that we would re-use the `eval_plan` when a new subscriber joins with the same query. If that's not the case, then either:
- We should do that.
- We should re-structure `ExecutionUnit` so that it isn't responsible for `eval`.
Either way, I think this should be a follow-up.